indico/web/flask/tasks.py

```diff
@@ -0,0 +1,23 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2024 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from datetime import datetime
+
+from celery.schedules import crontab
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.web.flask.models import IndicoStoredUserSession
+
+
+@celery.periodic_task(name='delete_stored_user_sessions', run_every=crontab(minute='0'))
+def delete_user_sessions():
+    """Delete expired user sessions from postgres."""
+    query = IndicoStoredUserSession.query.filter(IndicoStoredUserSession.ttl < datetime.now()).all()
```

Comment:
IIRC you can iterate directly on the query object:
```suggestion
    query = IndicoStoredUserSession.query.filter(IndicoStoredUserSession.ttl < datetime.now())
```

Though I think you can also call delete on the query:
```suggestion
    IndicoStoredUserSession.query.filter(IndicoStoredUserSession.ttl < datetime.now()).delete()
```